package paqueteRegistro;

public class ConsolaLogger implements Logger{
		
	public void registrar(String mensaje) {
		System.out.println(mensaje);
	}
	
	public void separador() {
		System.out.println("--------------");
	}
}
